cfme/tests/physical_infrastructure/test_physical_rack.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="function")]
+
+
+@pytest.fixture(scope="module")
+def physical_rack_collection(appliance, provider, setup_provider_modscope):
+    # Get and return the physical rack collection
+    return appliance.collections.physical_racks
```

Comment:
I don't think this should be in a fixture, its just the collection instance on the appliance, which can be accessed directly in any of the test functions. fixtures are for required setup or prerequisite resources for a test, which I don't think this falls into.